sdk/iot/Azure.Iot.Hub.Service/tests/prerequisites/setup.ps1

```diff
@@ -90,6 +90,7 @@ if (-not $hubExists)
     az iot hub create -n $IotHubName -g $ResourceGroup --location $Region --output none --only-show-errors
```

Comment:
We should be using ARM template, not az commands to create resources. It is a TODO for ADT. We did not do it initially because we did not know what works and what does not. Use the setup.ps1 in our old repo as example for IotHub. The ARM will also be repurposed for the pipeline later.

Reply:
I believe Azad is leading that effort. @azabbasi FYI

Reply:
You guys should sync and just do it once in that case. Let us not duplicate efforts.